Paintroid/src/androidTest/java/org/catrobat/paintroid/test/espresso/rtl/HindiNumberFormatTest.java

```diff
@@ -0,0 +1,93 @@
+package org.catrobat.paintroid.test.espresso.rtl;
+
+import android.support.test.rule.ActivityTestRule;
+import android.support.test.runner.AndroidJUnit4;
+
+import org.catrobat.paintroid.MainActivity;
+import org.catrobat.paintroid.R;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.util.Locale;
+
+import static android.support.test.espresso.Espresso.onData;
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.action.ViewActions.click;
+import static android.support.test.espresso.assertion.ViewAssertions.matches;
+import static android.support.test.espresso.matcher.ViewMatchers.withId;
+import static android.support.test.espresso.matcher.ViewMatchers.withText;
+import static org.catrobat.paintroid.test.espresso.util.EspressoUtils.clickColorPickerPresetSelectorButton;
+import static org.catrobat.paintroid.test.espresso.util.EspressoUtils.openColorPickerDialog;
+import static org.catrobat.paintroid.test.espresso.util.EspressoUtils.openNavigationDrawer;
+import static org.catrobat.paintroid.test.espresso.util.UiMatcher.withBackground;
+import static org.hamcrest.Matchers.allOf;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.hasToString;
+import static org.hamcrest.core.StringStartsWith.startsWith;
+
+@RunWith(AndroidJUnit4.class)
+public class HindiNumberFormatTest {
+    private static final Locale ARABICLOCALE = new Locale("ar");
```

Comment:
ident with tabs

Reply:
sorry! I do not understand what you mean with "ident with tabs".
please more explanation.

Reply:
sry, typo. I meant indent with tabs instead of spaces (for the whole java file)

You might have wrong settings in Android Studio:
in File->Settings->Editor->Code Style -> Java: check "Use tab characters"

And you can let Android Studio show the whitespaces
File->Settings->Editor-> General -> Appearance: check "Show whitespaces"